pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/JacksonConfigurator.java

```diff
@@ -33,6 +36,20 @@
 
     public JacksonConfigurator() {
         mapper = ObjectMapperFactory.create();
+        setInterfaceDefaultMapperModule();
+    }
+
+    private void setInterfaceDefaultMapperModule() {
+        SimpleModule module = new SimpleModule("InterfaceDefaultMapperModule");
+
+        // we not specific @JsonDeserialize annotation in OffloadProcessStatus
+        // because we do not want to have jackson dependency in pulsar-client-admin-api
+        // In this case we use SimpleAbstractTypeResolver to map interfaces to impls
+        SimpleAbstractTypeResolver resolver = new SimpleAbstractTypeResolver();
+        resolver.addMapping(OffloadProcessStatus.class, OffloadProcessStatusImpl.class);
```

Comment:
@merlimat I have added a `SimpleAbstractTypeResolver` here, and will manually map interfaces with impl classes here.